From c5442f9c7ce07720abd854d5e60580f173fdc0f0 Mon Sep 17 00:00:00 2001
From: Ulrich Hecht <uli@basicengine.org>
Date: Wed, 23 Aug 2023 17:38:56 +0200
Subject: [PATCH] reduce reserved memory size from 80 to 64MB

Seems to work just as well, even for 1920x1080. Might be possible to
reduce it further.
---
 cedar_ve.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cedar_ve.c b/cedar_ve.c
index 3348f54..d18dbce 100644
--- a/cedar_ve.c
+++ b/cedar_ve.c
@@ -1767,7 +1767,7 @@ static int cedardev_init(struct platform_device *pdev)
 #endif /*LINUX_VERSION_CODE >= KERNEL_VERSION(4,11,0)*/
 
 #if !defined(USE_ION)
-	cedar_devp->ve_size = 80 * SZ_1M;
+	cedar_devp->ve_size = 64 * SZ_1M;
 	dma_set_coherent_mask(cedar_devp->dev, 0xFFFFFFFF);
 	cedar_devp->ve_start_virt = dma_alloc_coherent(cedar_devp->dev, cedar_devp->ve_size,
 												   &cedar_devp->ve_start_pa,
-- 
2.39.2

